FirebaseAdmin/FirebaseAdmin.IntegrationTests/FirebaseMessagingTest.cs

```diff
@@ -124,78 +124,6 @@ public async Task SendEachForMulticast()
             Assert.NotNull(response.Responses[1].Exception);
         }
 
-        [Fact]
```

Comment:
Did you mean to remove this test?

Reply:
Yes. Because now `SendAllAsync` and `SendEachForMulticastAsync` have been marked as obsolete, these two integration tests will fail: https://github.com/firebase/firebase-admin-dotnet/actions/runs/5661155177/job/15338452380 .
I also tried marking these two tests with "[Ignore]", but it could not build. Any suggestions?